lib/Munin/Node/SpoolWriter.pm

```diff
@@ -16,10 +16,11 @@ use Munin::Common::Utils qw( is_valid_hostname );
 
 use Params::Validate qw( :all );
 use List::Util qw( first );
+use Readonly;
```

Comment:
Not sure I am a huge fan of that one as `constant` is a core feature.

Reply:
Far from knowing anything: `Readonly` seems to be a core library and `const` is a language feature, correct?

Reply:
Core library might be a little stretch. It's from CPAM.

Const is from a core lib. 

Nothing is a language feature in term of const-ness.

Reply:
> Core library might be a little stretch. It's from CPAN.

uh - OK - my misunderstanding of a local path.

In this case I would also tend to stick to the core feature set.

Reply:
@cgzones: can we stick to `const` or is there a good reason for `Readonly`?